MonticelloTonel-Core.package/TonelParser.class/instance/newMethodDefinitionFrom..st

```diff
@@ -15,5 +15,5 @@ newMethodDefinitionFrom: anArray
 		classIsMeta: meta
 		selector: selector
 		category: (metadata at: #category ifAbsent: [ '' ]) 
-		timeStamp: nil
+		timeStamp: (metadata at: #timestamp ifAbsent: [ nil ])
```

Comment:
Note that it is not only a timestamp, but also includes the author initials or author name.

The Tonel standard draft that I got proposes implementation-specific prefixes for properties, like gs_ for Gemstone and va_ for VA Smalltalk. In my review of the draft, I asked the author why these need to be so abbreviated. My question for this pull request is whether timestamps should be considered a Squeak feature or a general Smalltalk feature. I could not quickly  find a mention of method versions in "Smaltalk-80: The Interactive Programming Environment", so they might have been introduced with Squeak (and since been dropped by Pharo).

Reply:
In commentStamp there is also no 'time'.

Reply:
Both "timestamp" and "commentStamp" matches the existing Squeak vocabulary. Actually, "changeStamp" would be better for both cases. Still, that might not be what you are looking for.

Reply:
changeStamp sounds good to me and I'd rather go for that. Now the question remains whether it should be #changeStamp or #squeak_changeStamp or #sq_changeStamp?